atmo/users/migrations/0002_google_auth.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.1 on 2016-09-20 11:39
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def add_socialapp(apps, schema_editor):
+    SocialApp = apps.get_model('socialaccount', 'SocialApp')
+    Site = apps.get_model('sites', 'Site')
+    db_alias = schema_editor.connection.alias
+    app, created = SocialApp.objects.using(db_alias).get_or_create(
+        provider='google',
+        defaults={
+            'name': 'Google',
+            'client_id': '559124132137-h9jjj9jm7762uq7u2mn8h8dki7re28tv.apps.googleusercontent.com',
+            'secret': '1iNc8DKhNXoey8jLs4y-E6MN',
```

Comment:
I've replaced it with a make task and management command, thanks for the heads-up